src/components/tabs/tabs.tsx

```diff
@@ -33,7 +33,8 @@ export class Tabs {
   @Prop({ reflect: true }) layout: TabLayout = "inline";
 
   /**
-   * Display the tabs above (default) or below the tab content
+   * Display the tabs top (default) or bottom of the tab content. above and below are deprecated.
+   *
    */
   @Prop({ reflect: true }) position: TabPosition = "above";
```

Comment:
If above and below are deprecated should we set the default to be "top"?

Reply:
I wonder if we should use the @deprecated tag here to note the deprecation?

Reply:
if we use the @deprecated tag here, it will be applied to the prop rather than values.

Reply:
Agree that we need to make `top` the default here for tabs.